embed/ephy-embed.c

```diff
@@ -29,6 +29,7 @@
 #include "ephy-embed-shell.h"
 #include "ephy-embed-utils.h"
 #include "ephy-find-toolbar.h"
+#include "ephy-notification-manager.h"
```

Comment:
Ah, this is why you added the password notification files to embed: you need them here in EphyEmbed.

I was thinking it might be better to avoid this dependency by placing the GtkOverlay on the EphyWindow instead of the EphyEmbed, but we always want the notification to appear over the embed, so I think your current code is best after all. This means you should disregard my previous comments about moving the password notification files to src. (But you could probably still move ephy-fx-password-notification to lib/widgets, as libephywidgets.la gets linked into libephyembed.la. The stuff under lib/widgets can be used in the embed directory, even though the stuff in the toplevel lib directory cannot.)